graph_weather/data/nnjai_wrapp.py

```diff
@@ -0,0 +1,106 @@
+"""
+A custom PyTorch Dataset implementation for various sensors like AMSU, ATMS, MHS, IASI, CrIS 
+
+The dataset is loaded via the nnja library's `DataCatalog` and filtered for specific times and
+variables. Each data point consists of a timestamp, latitude, longitude, and associated metadata.
+"""
+
+import numpy as np
+import torch
+from torch.utils.data import Dataset
+
+try:
+    from nnja import DataCatalog
+except ImportError:
+    print(
+        "NNJA-AI library not installed. Please install with `pip install git+https://github.com/brightbandtech/nnja-ai.git`"
+    )
+
+
+class SensorDataset(Dataset):
```

Comment:
```suggestion
class NNJADataset(Dataset):
```

We might (probably will, as I am looking into adding other, non NNJA sensors) have other sensors that won't fit in this dataset. I think naming this just `NNJADataset` then is more descriptive and easier to parse where this data is coming from.